### PR TITLE
WINUI-2993 Update chart API changes and code alignment

### PR DESCRIPTION
**Pull request by naveenkumar-sanjeevirayan**
### Feature description ###

Update new API changes in the WinUI chart controls.

### Analysis and design ###

[If there is an external design, link to its project documentation area.
If there is an internal discussion on the forum, provide the link.](https://syncfusion.sharepoint.com/sites/XamarinChart/_layouts/15/Doc.aspx?sourcedoc={4d149321-98e7-41ca-a4c9-9ba91b394ac3}&action=edit&wd=target%28WinUI%20Chart.one%7C6e982125-140d-45fb-b696-9c12636c7f7c%2FWinUI%20Chart%20API%20Breaking%20Changes%20Review%7Cbae4d30c-59b1-4bd7-a948-6a6b07427f40%2F%29&wdorigin=NavigationUrl)

### Solution description ###

Based on the below API changes for the WinUI chart, updated chart API changes and code alignment for cartesian, circular, funnel, pyramid, and polar charts.

### Output screenshots ###

NA

### Areas affected and ensured ###

NA

### API Changes ###

API breaking changes,
https://syncfusion.sharepoint.com/sites/XamarinChart/_layouts/15/Doc.aspx?sourcedoc={4d149321-98e7-41ca-a4c9-9ba91b394ac3}&action=edit&wd=target%28WinUI%20Chart.one%7C6e982125-140d-45fb-b696-9c12636c7f7c%2FWinUI%20Chart%20API%20Breaking%20Changes%20Review%7Cbae4d30c-59b1-4bd7-a948-6a6b07427f40%2F%29&wdorigin=703

### MR CheckList ###

- [ ] Does it follow the design [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)? It is mandatory that, we should not move the changes without reading this. **NA**
- [ ] Did UI automation passed without errors? If it is customer issue, make sure it is included in the IR automation. **NA**
- [ ] Properly working in Xamarin.Forms [previewer](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/xaml/xaml-previewer?tabs=vswin). **NA**
- [ ] Ensured in iOS, Android, UWP and macOS (if supported). **NA**
- [ ] Have you added API comments if it is new API?  **NA**
- [ ] Does it need localization? If so, update resx file in [GitHub](https://github.com/syncfusion/xamarin-localized-texts).**NA**
- [x] Whether we have added new APIs? If so, whether it is as per the [guidelines](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/)?
- [ ] If changes are made in the Xamarin.Forms sample browser project file, whether it is included in the props file too? If not, your sample level changes will not reflect in the application published in the hockey app. **NA**
- [ ] Have you ensured the changes in Android API 19 and iOS 9? **NA**

### CODE REVIEW

File: winui/Cartesian-Charts/Axis/Axislabels.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Axis labels are used to show the units, measures, or category value of the axis
 
 ## Rotation Angle
 
-The [LabelRotationAngle](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartAxis.html#Syncfusion_UI_Xaml_Charts_ChartAxis_LabelRotationAngle) property is used to define the angle for the label content.
+The [LabelRotation](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartAxis.html#Syncfusion_UI_Xaml_Charts_ChartAxis_LabelRotationAngle) property is used to define the angle for the label content.
```
Comment (MuneeshKumarG) [inline winui/Cartesian-Charts/Axis/Axislabels.md:17]: @naveenkumar-sanjeevirayan  : Remove the old links

File: winui/Pyramid-Chart/Selection.md [hunk 2]
```diff
@@ -192,15 +192,15 @@ The following selection events are available in the [ChartSelectionBehavior](htt
...
 * [Cancel](https://docs.microsoft.com/en-us/dotnet/api/system.componentmodel.canceleventargs.cancel?view=net-6.0) - Gets or sets a value that indicates whether the selection should be canceled.
 
 ### SelectionChanged
 
-The [SelectionChanged](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartBase.html#Syncfusion_UI_Xaml_Charts_ChartBase_SelectionChanged) event occurs after a data point has been selected. This argument contains the following information.
+The [SelectionChanged](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartBase.html#Syncfusion_UI_Xaml_Charts_ChartBase_SelectionChanged) event occurs after a data point has been selected. The `ChartSelectionChangedEventArgs` contains the following information.
 
 * [CurrentIndex](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SelectionChangedEventArgs.html#Syncfusion_UI_Xaml_Charts_SelectionChangedEventArgs_CurrentIndex) - Gets the selected data point index.
```
Comment (MuneeshKumarG) [inline winui/Pyramid-Chart/Selection.md:205]: CurrentIndex -> NewIndexes[0]
PreviousIndex -> OldIndexes[0]

Comment (naveenkumar-sanjeevirayan) [reply winui/Pyramid-Chart/Selection.md]: I have updated the above API changes.

File: winui/Cartesian-Charts/Axis/Axislabels.md (hunk 1 above)
Comment (naveenkumar-sanjeevirayan) [reply winui/Cartesian-Charts/Axis/Axislabels.md:17]: @MuneeshKumarG , I have removed the old website links.

**Comment by naveenkumar-sanjeevirayan** [comment]
rebuild

File: winui/Cartesian-Charts/Selection.md [hunk 3]
```diff
@@ -145,7 +145,7 @@ this.Content = chart;
...
+Cartesian chart provides support to select single or multiple segments/series by using the [Type](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionBehavior.html#Syncfusion_UI_Xaml_Charts_ChartSelectionBehavior_Type) property. [Type](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionBehavior.html#Syncfusion_UI_Xaml_Charts_ChartSelectionBehavior_Type) property with [Multiple]() value is used to select multiple segments/series. By default, the [Type](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartSelectionBehavior.html#Syncfusion_UI_Xaml_Charts_ChartSelectionBehavior_Type) value is [Single]().
```
Comment (MuneeshKumarG) [inline winui/Cartesian-Charts/Selection.md:149]: @naveenkumar-sanjeevirayan  : Add properties in top, refer below PR

https://github.com/syncfusion-content/maui-docs/pull/534/files

Comment (naveenkumar-sanjeevirayan) [reply winui/Cartesian-Charts/Selection.md:163]: @MuneeshKumarG , I have added properties topic in selection files.

**Comment by naveenkumar-sanjeevirayan** [comment]
rebuild

**Pull request merged**